tasks/sast-go.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: gosec-check
```

Comment:
Please use same name of task as name of the file.

Reply:
Done